elephant/statistics.py

```diff
@@ -524,3 +525,101 @@ def instantaneous_rate(spiketrain, sampling_period, form, sigma, m_idx=None,
                                  units=pq.Hz, t_start=t_start)
 
     return rate
+
+
+def psth(spiketrains, binsize, t_start=None, t_stop=None, output='counts',
+         clip=False):
```

Comment:
I think that we should move towards recognising that this is really a delta function representation, either Dirac or Kronecker depending on ... well you know this.  So maybe we could call the option 'delta', or 'delta_clip'?  Perhaps we could also use the same terminology in the binned spike train class?